src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,6 +7,7 @@ public class CliSyntax {
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_AGE = new Prefix("A/");
```

Comment:
Would this use of "A/" for age and "a/" for address be confusing for the user? Are there any alternatives you considered?

Reply:
Agreed, might be confusing to the user. Perhaps using "a/" for age and "ad/" for address would cause less confusion? What do you guys think? :)

Reply:
It may be possible to rename it to b/? This is in response to my other comment on the Age class as well.

Reply:
Yes, "b/" for birthdate definitely sounds like a good idea!